dns_sd_windows.c

```diff
@@ -0,0 +1,262 @@
+// Based on the example provided here: https://github.com/mjansson/mdns
+
+#ifdef _WIN32
+#  define _CRT_SECURE_NO_WARNINGS 1
```

Comment:
We specifically do not do this anymore. Files which are Windows, compile with the secure strings settings